pkg/drain/evictor/pod.go

```diff
@@ -204,7 +203,7 @@ func (d *Evictor) daemonSetFilter(pod corev1.Pod) PodDeleteStatus {
 		if controllerRef.Name == ignoreDaemonSet.Name {
 			switch ignoreDaemonSet.Namespace {
 			case pod.Namespace, metav1.NamespaceAll:
-				return makePodDeleteStatusWithWarning(false, daemonSetWarning)
+				return makePodDeleteStatusSkip()
```

Comment:
This error is super noisy and adds next to zero value.

There may be a better way to disable it / filter it out but this was the best I found

Reply:
Yeah, that error came from some copied over eviction code from 3 years ago. I don't think we need it... :D